CIP-????/README.md

```diff
@@ -0,0 +1,67 @@
+---
+CIP: 101
```

Comment:
```suggestion
CIP: ?
```
@paluh I can see you got permission to copy some of the text... but it's not helpful to copy this particular item 😅

Reply:
:-)